Tests/ValetIntegrationTests/ValetIntegrationTests.swift

```diff
@@ -142,33 +144,6 @@ class ValetIntegrationTests: XCTestCase
         }
     }
 
-    // MARK: Equality
```

Comment:
These tests already exist in `ValetTests.swift`. I think I mistakenly left them in both files when I split this file in to two for SPM.